src/components/TankSkeletons/AllyTankSkeleton.tsx

```diff
@@ -4,7 +4,7 @@ import { useGridObjectState } from '../Grid/Grid.helpers';
 
 const AllyTankSkeleton: FC = () => {
   const [, inlineStyles] = useGridObjectState<undefined>();
-  return <div style={inlineStyles} className={[styles.AllyTankSkeleton, styles.TankSkeleton].join(' ')} />;
+  return <div style={inlineStyles} className={[styles.TankSkeleton, styles.AllyTankSkeleton].join(' ')} />;
```

Comment:
- [ ] I suggest moving `[styles.TankSkeleton, styles.AllyTankSkeleton].join(' ')` to a separate variable for clearance.